commands/deploy.go

```diff
@@ -276,7 +276,9 @@ Error: %s`, fprocessErr.Error())
 			if msg := checkTLSInsecure(services.Provider.GatewayURL, deploySpec.TLSInsecure); len(msg) > 0 {
 				fmt.Println(msg)
 			}
-			statusCode := proxyClient.DeployFunction(ctx, deploySpec)
+			statusCode, deployOutputStr := proxyClient.DeployFunction(ctx, deploySpec)
```

Comment:
I think this is better, but it's not what we discussed on last week's members call. We agreed to refactor to a pattern more similar to the GitHub go SDK where a Go struct, HTTP response,  and error are returned from the call.